src/darwin_universal_binary.bzl

```diff
@@ -0,0 +1,54 @@
+def _universal_split_transition_impl(ctx, attr):
+    return {
+        "x86_64" : {
```

Comment:
This doesn't support platformization, which we'll need to update when we turn it on.

Reply:
Please also set `--platforms` flag to match the cpu. This then won't cause problems when we flip `--incompatible_enable_cc_toolchain_resolution`